libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -731,7 +736,7 @@ bool ASTJsonConverter::visit(FunctionCall const& _node)
 		attributes.emplace_back("isStructConstructorCall", _node.annotation().kind == FunctionCallKind::StructConstructorCall);
 		attributes.emplace_back("type_conversion", _node.annotation().kind == FunctionCallKind::TypeConversion);
 	}
-	else
+	else if (_node.annotation().kind != FunctionCallKind::Unset)
```

Comment:
Is this a bugfix?

Reply:
Well, it fixes a bug that happened as a result from the stop-after-parse change. I think usually you wouldn't get `Unset`.